SetupBLT.cmake

```diff
@@ -187,7 +187,9 @@ if (NOT BLT_LOADED)
     # File extension lists used to filter sources based on languages for code checks
     # and filtering Fortran sources out of cuda/hip source lists
     # Note: this filtering is case-insensitive
-    set(BLT_C_FILE_EXTS ".cpp" ".hpp" ".cxx" ".hxx" ".c" ".h" ".cc" ".hh" ".inl" ".cu" ".cuh"
+    set(BLT_C_FILE_EXTS ".c" ".h"
+               CACHE STRING "List of known file extensions used for C/CXX sources")
```

Comment:
Update the comment to say C rather than C/CXX. And the following comment should have CXX instead of C/CXX.

Reply:
This will have pretty large user facing consequences and needs to be considered carefully.

Reply:
Might only be my problem though:

https://sourcegraph.com/search?q=context:global+repo:github.com/llnl/*+BLT_C_FILE_EXTS&patternType=standard